slack-connector/README.md

```diff
@@ -0,0 +1,6 @@
+# Slack Connector
+
+## Overview
+
+Slack Connector is a component which allows contact from inside of Kyma environment to the Slack API.\
+It registers an application in the Application Registry. The installation guide can be found [here](/docs/slack-connector/README.md)
```

Comment:
```suggestion
It registers an application in the Application Registry. The installation guide can be found [here](/docs/slack-connector/README.md)
```
The guidelines say you shouldn't use passive voice (although I do agree that `the installation guide can be found here` sounds good).
Maybe rephrase that to something like:
> See [the installation guide](/docs/slack-connector/README.md).

or 

> See the installation guide [here](/docs/slack-connector/README.md).

?